### PR TITLE
 2021 03 26 vasilemaftita friday coding

### PR DESCRIPTION
**Pull request by vasilemaftita**

### CODE REVIEW

**Review by Constantin-Matei — APPROVED** [review]

**Review by RalucaRbc — APPROVED** [review]
💯

**Review by MariaTeo — APPROVED** [review]
nice work

File: web/vasilemaftita/html/formatting/index.html [hunk 1]
```diff
@@ -13,4 +13,4 @@
 <sup>Do you want to go?</sup></p>
     
 </body>
-</html>
\ No newline at end of file
+</html>
```
Comment (marianzburlea) [inline web/vasilemaftita/html/formatting/index.html:14]: missing starting tag `<p>` on line 13

**Review by marianzburlea — DISMISSED** [review]
add the starting p tag

**Review by QuintyHH — APPROVED** [review]
:100: once you update the code as Marian requested.

**Pull request merged**